test/e2e/map-modify-actions.test.js

```diff
@@ -0,0 +1,73 @@
+const {assert, getDriver} = require('vl-ui-core').Test.Setup;
+const VlMapModifyActionsPage = require('./pages/vl-map-modify-actions.page');
+
+describe('vl-map-modify-action', async () => {
+  let vlMapPage;
+  const willebroek = [150390.63, 193371.12];
+
+  before(() => {
+    vlMapPage = new VlMapModifyActionsPage(getDriver());
+    return vlMapPage.load();
+  });
+
+  it('als gebruiker zie ik dat de aanpas actie actief staat', async () => {
+    const action = await vlMapPage.getModifyPointAction();
+    await assert.eventually.isTrue(action.isActive());
+  });
+
+  it('als gebruiker kan ik een punt aanpassen op een kaart', async () => {
+    const action = await vlMapPage.getModifyPointAction();
```

Comment:
Best eerst de map opvragen aangezien deze vermoedelijk voor meer stabiliteit zal zorgen via de ready functionaliteit. De modify action kan dan via de map opgevraagd worden.

Reply:
Als ge dat doet hebt ge weer circulair he. Zegt ge tegen een actie "doe iets" en die haalt dan de VlMap op, kunt ge niet nu via de VlMap een actie ophalen he

Reply:
Nee, ik bedoel dat je in de e2e testen altijd vertrekt vanuit je map:

```
const map = await page.getMap();
const action = await map.findElement(By.css('map')); // move me to page object
```